builtin/logical/pki/path_issue_sign.go

```diff
@@ -222,7 +222,7 @@ func (b *backend) pathIssueSignCert(
 	var resp *logical.Response
 	switch {
 	case role.GenerateLease == nil:
-		return nil, fmt.Errorf("Genereate lease in role is nil")
+		return nil, fmt.Errorf("genereate lease in role is nil")
```

Comment:
s/genereate/generate -- should probably also make it generate_lease with an underscore to match the param.